donations/urls.py

```diff
@@ -26,6 +26,8 @@
 urlpatterns = [
     url(r'^donate/$', views.donate, name="donate"),
     url(r'^donors/$', views.DonationsList.as_view(), name="donors"),
+    url(r'^donation-session-stripe/$', views.donation_session_stripe, name="donation-session-stripe"),
+    url(r'^donation-success-stripe/$', views.donation_success_stripe, name="donation-success-stripe"),
```

Comment:
donation success is for when user comes back from stripe page (after making the donation) but still we have not received the webhook callback from stripe right?

for donating with paypal we also go to the paypal website and then go back to Freesound right? or how does it work? If the flows are similar it would be great to make sure all views are named following the same structure.

Reply:
Yes, you're right. Donation success is where the user is redirected, it only shows a message saying that the donation will be processed and redirects to the donations page.

For Paypal the url 'donation-complete-paypal' also listens to paypal requests and creates the Donation object. Therefore, now the names are more consistent than before.